rdt-open-reader/src/covidIndonesia/assets/json.form/patient-registration-form-original.json

```diff
@@ -273,6 +273,23 @@
         "openmrs_entity_id": "",
         "type": "edit_text"
       },
+      {
+        "key": "gvt_id_address_lbl",
```

Comment:
Could the fields be renamed to  have `gvt_id_street_address` instead of `gvt_id_address`.

Reply:
fields id updated.